modules/client/src/multisig/internal/client/decoding.ts

```diff
@@ -25,39 +26,47 @@ export class MultisigClientDecoding extends ClientCore
   /**
    * Decodes a list of addresses from an encoded add members action
    *
-   * @param {Uint8Array} data
-   * @return {*}  {string[]}
+   * @param {Uint8Array[]} data
+   * @return {*}  {MultisigPluginSettings}
    * @memberof MultisigClientDecoding
    */
-  public addAddressesAction(data: Uint8Array): string[] {
+  public addAddressesAction(data: Uint8Array[]): MultisigPluginSettings {
```

Comment:
This type may lead to confusion. An array of addresses conveys that these addresses have been added.

But a `MultisigPluginSettings` object conveys that this is the final state of the multisig (including the existing wallets)